src/item.cpp

```diff
@@ -3161,7 +3118,7 @@ units::volume item::volume( bool integral ) const
     }
 
     if( is_corpse() ) {
-        return corpse_volume( corpse->size );
+        return corpse_volume( corpse->volume );
```

Comment:
I suggest getting rid of this parameter and using `corpse->volume` inside that function directly. It already accesses other properties of the `item` and having this as separate parameter just calls for inconsistency (like calling it with the volume of some other corpse item).

Reply:
i don't understand what you're saying. `corpse_volume()` is a helper function that's used in two different places that modifies it based on a couple of flags created by butchery

Reply:
If you have access to `corpse->volume` inside of `corpse_volume` function there is no need to pass it as function parameter.